ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
Arg....I now see how I broke it with the AVPixelFormat.  It is such fun making everything compatible for eight or so versions and variations at the same time.